ocfweb/docs/docs/staff/procedures/ssh-supernova.md

```diff
@@ -0,0 +1,51 @@
+[[!meta title="SSHing into Supernova"]]
+
+When contributing to the OCF codebase, you will need some way of running and
+testing your code. Most of our applications cannot be run on a personal
+machine-- they need to be run on OCF infrastructure. For most projects we
+recommend doing all development from `supernova`, a server available to all
+staff. Our [[Kubernetes server|doc staff/backend/kubernetes]] is also
+accessible from `supernova`.
+
+## Before you begin
+You won't be able to log into the server if you're on staff. If you're not sure
+whether or not you're on staff, try logging in. If this doesn't work, talk to a
+staff member and we can add you! All are welcome and encouraged to join staff.
+
+## Logging in
+To log in, open a terminal window and type in:
+
+```
+ssh username@supernova.ocf.berkeley.edu
```

Comment:
maybe `<username>`?

Reply:
This is inconsistent with other parts of our docs, and I prefer to not do that anyways, since I sometimes see people misinterpret this and include the angle brackets when they re-type the command. I think it's pretty clear what's going on here.